soroban-env-host/src/fees.rs

```diff
@@ -261,7 +266,11 @@ pub fn compute_write_fee_per_1kb(
         .clamp_fee();
         write_fee_per_1kb = write_fee_per_1kb.saturating_add(post_target_fee);
     }
-    write_fee_per_1kb
+    if write_fee_per_1kb < MINIMUM_WRITE_FEE_PER_1KB {
+        MINIMUM_WRITE_FEE_PER_1KB
+    } else {
+        write_fee_per_1kb
+    }
```

Comment:
nit: you can just do `write_fee_per_1kb.max(MINIMUM_WRITE_FEE_PER_1KB)`

Reply:
Good point. Fixed